torch_lr_finder/lr_finder.py

```diff
@@ -414,7 +415,7 @@ def _validate(self, val_iter, non_blocking_transfer=True):
 
         return running_loss / len(val_iter.dataset)
 
-    def plot(self, skip_start=10, skip_end=5, log_lr=True, show_lr=None, ax=None):
+    def plot(self, skip_start=10, skip_end=5, log_lr=True, show_lr=None, ax=None, suggestion=False):
```

Comment:
We should change the name to something more descriptive, like `suggest_lr`

Reply:
```suggestion
    def plot(self, skip_start=10, skip_end=5, log_lr=True, show_lr=None, ax=None, suggest_lr=None):
```

Reply:
I would actually suggest keeping `suggest_lr` as a `bool` since there's only one algorithm for suggesting a learning rate. I don't know when or if more will be implemented and in the meantime having it as an `str` might be confusing for users if there's only one choice.